gc/base/standard/ConcurrentSweepScheme.cpp

```diff
@@ -722,18 +723,19 @@ MM_ConcurrentSweepScheme::setupForSweep(MM_EnvironmentBase *env)
 void
 MM_ConcurrentSweepScheme::verifyFreeList(MM_EnvironmentStandard *env, MM_HeapLinkedFreeHeader *freeListHead)
 {
-	for (MM_HeapLinkedFreeHeader *current = freeListHead; NULL != current; current = current->getNext()) {
+	bool const compressed = env->compressObjectReferences();
+	for (MM_HeapLinkedFreeHeader *current = freeListHead; NULL != current; current = current->getNext(compressed)) {
 		assume(current < current->afterEnd(), "Free list size overflows");
-		assume( (current->getNext() == NULL) || (current < current->getNext()), "Free list next pointer is lower in memory");
-		assume( (current->getNext() == NULL) || (current->getNext() > current->afterEnd()), "Size is too large (flows into next free entry)");
+		assume( (current->getNext(compressed) == NULL) || (current < current->getNext(compressed)), "Free list next pointer is lower in memory");
+		assume( (current->getNext(compressed) == NULL) || (current->getNext(compressed) > current->afterEnd()), "Size is too large (flows into next free entry)");
 
 #if 1
```

Comment:
Would you please remove this `#if 1` as well ?

Reply:
Done.